recipe/thirdparty/utils.go

```diff
@@ -95,6 +100,14 @@ func parseUsers(value interface{}) ([]tpmodels.User, error) {
 	}
 	var user []tpmodels.User
 	err = json.Unmarshal(respJSON, &user)
+
```

Comment:
why not just call the parseUser function in a loop in this function?

Reply:
Done